scripts/rabbitmq-server

```diff
@@ -118,6 +118,7 @@ start_rabbitmq_server() {
         -sasl sasl_error_logger "$SASL_ERROR_LOGGER" \
         -rabbit error_logger "$RABBIT_ERROR_LOGGER" \
         -rabbit sasl_error_logger "$RABBIT_SASL_ERROR_LOGGER" \
+        -rabbit log_base "\"$RABBITMQ_LOG_BASE\"" \
```

Comment:
if we add this in `rabbitmq-server`, we need to do the same in `rabbitmq-server.bat`.